bin/git-create-branch

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env bash
 
+# preference takes lowest priority; look for remote from prefs first
+REMOTE_PREF=`git config git-extras.create-branch.remote`
+if [ "$REMOTE_PREF" == "true" ]; then
+	REMOTE=origin
```

Comment:
A little question: Why we need to use `git config git-extras.create-branch.remote true` to indicate the `origin` remote? IMHO, using `git config git-extras.create-branch.remote origin` is enough, which is simpler.

Reply:
@spacewander , fair point. My intention was to make the pref in direct parity with the argument of the command.  Presence of the `[-r|--remote]` option with no argument acts as a boolean. Perhaps parity adds awkwardness in this case. I'm happy to take your suggestion.  Knowing my intent, let me know what your thought is on that.

Reply:
@btmurrell 
IMHO, if people want to set their preference, they will not want to use the default remote defined by the tool author. So I think we don't need to preserve a default option for the preference. What about your idea?

Reply:
you're right. that's a more common sense way to use it.